pkgs/top-level/all-packages.nix

```diff
@@ -11624,6 +11624,7 @@ in
   bingrep = callPackage ../development/tools/analysis/bingrep { };
 
   binutils-unwrapped = callPackage ../development/tools/misc/binutils {
+    autoreconfHook = if targetPlatform.isiOS then autoreconfHook269 else autoreconfHook;
```

Comment:
This is annoying! When do you need to build binutils-unwrapped?

Reply:
It seems to get built as part of the local bootstrapping procedure after you’ve fulfilled the XCode dependency by adding it to the store. If you build the hello app you get the error, it’s pretty reproducible.

Reply:
@matthewbauer I think for the other tools beyond the cctools ones.